webull.py

```diff
@@ -433,8 +466,15 @@ def cancel_otoco_order(self, order_id=''):
     '''
     get price quote
     '''
-    def get_quote(self, stock='') :
-        response = requests.get(urls.quotes(self.get_ticker(stock)))
+    def get_quote(self, stock=None, tId=None) :
+        if not stock is None:
+            tId = self.get_ticker(stock)
+        elif not tId is None:
+            tId = str(tId)
```

Comment:
format strings should have no problem with an integer, no need to convert to string here. 
Case not needed just to do tId = tId then.

Reply:
Roger, thks for feedback